relay-sampling/Cargo.toml

```diff
@@ -19,6 +19,7 @@ relay-filter = { path = "../relay-filter" }
 rand = "0.6.5"
 rand_pcg = "0.1.2"
 unicase = "2.6.0"
+chrono = "0.4.11"
```

Comment:
This is the version used in other packages (`relay-auth`, `relay-cabi`, `relay-common`...).